models/java/README.md

```diff
@@ -0,0 +1,84 @@
+# gbfs-java-model
+
+![Maven Central Version](https://img.shields.io/maven-central/v/org.mobilitydata.gbfs/gbfs-java-model.svg)
+
+Generates Java model from GBFS json schema using jsonschema2pojo with jackson2 annotations.
+
+In order to avoid naming conflicts in generated classes, each feed gets its own package, and all
+classes are prefixed with "GBFS".
+
+Packages include GBFS versions to accommodate for future releases which may include older versions,
+since some consumers may need to handle multiple non-compatible versions in the same application 
+(i.e. you should be able to parse v2.1 feeds with the v2.2 package, but v3.0 package would not
+be able to parse v2.2 feeds).
+
+## Add the Dependency 
+
+To use the `gbfs-java-model` classes in your own project, you need to add
+an appropriate dependency.  We publish our module to the [Maven Central Repository](http://search.maven.org/)
+so that it can be easily referenced by Java build tools like Maven, Ivy, and Gradle.
+
+For [Maven](http://maven.apache.org/), add the following to your `pom.xml`
+dependencies section:
+
+```xml
+<dependency>
+  <groupId>org.mobilitydata</groupId>
+  <artifactId>gbfs-java-model</artifactId>
+  <version>1.0.0</version>
```

Comment:
we're going to want to release at `3.1.3` and not `1.0.0` since it's a new package?

Reply:
Common practice to use a SNAPSHOT version between releases, so the version in the main branch is always the next version snapshot. E.g current version is 3.1.3 then version in main branch will be 3.1.4-SNAPSHOT. 

Now you can choose to continue on the versioning from Entur or start over from 1.0.0. I think there are valid arguments for both options. That said, since package paths are changed, library users will have to do a code migration to a simple groupId change is not enough. That might suggest either start new versioning or at least a new major version.